src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -102,5 +102,15 @@ struct TraceExtrinsicCurvature : db::SimpleTag {
   using type = Scalar<DataType>;
   static std::string name() noexcept { return "TraceExtrinsicCurvature"; }
 };
+
+/*!
+ * \brief The energy density \f$\rho=n_a n_b T^{ab}\f$
```

Comment:
`rho` is the rest mass density in the hydro equations, please use `e` to denote the energy density